LongWarOfTheChosen/Localization/LW_PerkPack_Integrated/LW_PerkPack_Integrated.rus

```diff
@@ -43,3 +43,125 @@ DenseSmokeGrenadeEffectDisplayDesc="Густой дым дополнительн
 
 [X2Ability_PerkPackAbilitySet]
 LocCoveringFire="Огневая поддержка"
+
+; LWOTC Translated
+LocPromotionPopupText="<Bullet/> Указанный враг получает -20 к точности и -50 и шансу крита к этому оперативнику.<br/><Bullet/> Свободное действие.<br/><Bullet/> Щит имеет <Ability:SelfCooldown/> ходов перезарядки."
+; End Translated
+
+[X2Ability_PerkPackAbilitySet]
+LocCoveringFireMalus="По этому юниту был произведен выстрел из режима наблюдения со способностью \"Огневая поддержка\", дающий штраф -10 к меткости до конца хода."
+
+
+; LWOTC Translated
++AbilityTreeTitles[0]="Битва Будущего"
++AbilityTreeTitles[1]="Непробиваемость"
++AbilityTreeTitles[2]="Машина войны"
++AbilityTreeTitles[3]="XCOM"
+; End Translated
+
+; LWOTC Translated
+LocLongDescription="У вас есть +<Ability:TRIGGER_BSC_AIM_BONUS/> (после штрафа) попасть ответным огнём с основого оружия."
+; End Translated
+
+; LWOTC Translated
+LocLongDescription="У вас есть +<Ability:TRIGGER_ADV_AIM_BONUS/> (после штрафа) попасть ответным огнём с основого оружия."
+; End Translated
+
+; LWOTC Translated
+LocLongDescription="У вас есть +<Ability:TRIGGER_SUP_AIM_BONUS/> (после штрафа) попасть ответным огнём с основого оружия."
+; End Translated
```

Comment:
These three need to be moved to the *Localization/XComGame.int* file under and placed under the `[Hair_Trigger_LW_Bsc_Ability X2AbilityTemplate]` section header.

Reply:
.int? nor rus?

Reply:
I`ll put in .rus 
think it was mistake

Reply:
Also there was already original lw2 translation. *it was not need to translate* (

Reply:
Yes, sorry, I meant move to *XComGame.rus*.